src/serverMain/kotlin/com/studo/campusqr/endpoints/QrCodes.kt

```diff
@@ -46,7 +46,10 @@ suspend fun AuthenticatedApplicationCall.viewSingleQrCode() {
             ).get(this@viewSingleQrCode)
           }
         }
-        renderLocation(location, configs)
+        div {
+          id = "allcodes"
```

Comment:
here and everywhere: allcodes is not a single word. I think you mean allCodes ;)